cumulus/parachains/runtimes/assets/asset-hub-rococo/tests/tests.rs

```diff
@@ -38,6 +38,7 @@ use codec::{Decode, Encode};
 use core::ops::Mul;
 use cumulus_primitives_utility::ChargeWeightInFungibles;
 use frame_support::{
+	__private::sp_io,
```

Comment:
```suggestion
```

Reply:
then add `sp_io` to cargo.toml in dev-dependencies

importing from `__private` submodule of any crate is an anti-pattern and can lead to breakages down the line, `__private` module of a crate can be backwards-incompatibly changed under the assumption that it is _private_ and NOT used by downstream crates.